tests/fdb/daos/test_daos_store.cc

```diff
@@ -317,8 +319,8 @@ CASE("DaosStore tests") {
             fdb5::TocCatalogueWriter tcat{db_key, config};
             fdb5::Catalogue& cat        = static_cast<fdb5::Catalogue&>(tcat);
             metkit::mars::MarsRequest r = db_key.request("retrieve");
-            std::unique_ptr<fdb5::WipeVisitor> wv(cat.wipeVisitor(store, r, out, true, false, false));
-            cat.visitEntries(*wv, false);
+            // std::unique_ptr<fdb5::WipeVisitor> wv(cat.wipeVisitor(r, queue, /*out,*/ true, false, false));
```

Comment:
This looks like a hack to make a test look like it's passing. That is dangerous. I'd rather disable the test than that.